pnpm-lock.yaml

```diff
@@ -4,6 +4,7 @@ specifiers:
   '@commitlint/cli': ^17.0.1
   '@commitlint/config-conventional': ^17.0.0
   '@fastify/env': ^3.0.0
+  '@prisma/client': ^3.14.0
```

Comment:
Did u apply rebase. Coz these were already added before. There should not be any change in lock file as ur not installing any new library

Reply:
before this prisma was throwing some error ! also what should i do to go back to previous state in this file ?